flow-typed/types/ui-options.js

```diff
@@ -8,5 +8,8 @@ declare type UIOptionsObject = {
   components?: ComponentsConfig,
   uiComponents?: Array<PKUIComponent>,
   translations?: {[langKey: string]: Object},
-  locale?: string
+  locale?: string,
+  disable?: boolean,
+  css?: string,
```

Comment:
these two are actually only part of kaltura player ui-wrapper.
If we would to export types then we would have inherited from them in wrapper, and add these two there